docs/fundamentals/syslib-diagnostics/syslib1230-1239.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SYSLIB diagnostics for Microsoft.Interop.COMInteropGenerator in .NET 9 and later
+description: Learn about the COM interop source-generation analyzers that generate compile-time suggestions SYSLIB1230 through SYSLIB1239.
+ms.date: 10/08/2024
+f1_keywords:
+  - syslib1230
+  - syslib1231
+  - syslib1232
+  - syslib1233
+  - syslib1234
+  - syslib1235
+  - syslib1236
+  - syslib1237
+  - syslib1238
+  - syslib1239
+---
+# SYSLIB diagnostics for COM interop source generation in .NET 9 and later
+
+The following table shows the diagnostic IDs for COM interop source-generation analyzers in .NET 9 and later versions.
+
+| Diagnostic ID | Description |
+| - | - |
+| `SYSLIB1230` | Deriving from a `GeneratedComInterface`-attributed interface defined in another assembly is not supported. |
+
+[!INCLUDE [suppress-syslib-warning](includes/suppress-source-generator-diagnostics.md)]

```

Comment:
If we keep in separate files (which I'm not crazy about) we should at least link between them in the # See also section.